be/src/exec/pipeline/fragment_context.h

```diff
@@ -174,6 +175,10 @@ class FragmentContext {
     // acquire runtime filter from cache
     void acquire_runtime_filters();
 
+    bool enable_event_scheduler() const { return event_scheduler() != nullptr; }
```

Comment:
nit: shall it be `event_scheduler_enabled`? `enable_event_scheduler` reads like to perform an action, turn on the event_scheduler, not query the on/off status.

Reply:
our session variables style is enable_xxx_feature